CHANGELOG.rst

```diff
@@ -0,0 +1,20 @@
+Change Log
+==========
+
+..
+   All enhancements and patches to cookiecutter-django-app will be documented
+   in this file.  It adheres to the structure of http://keepachangelog.com/ ,
+   but in reStructuredText instead of Markdown (for ease of incorporation into
+   Sphinx documentation).
+   
+   This project adheres to Semantic Versioning (http://semver.org/).
+
+.. There should always be an "Unreleased" section for changes pending release.
+
+Unreleased
+----------
+
+Added
+~~~~~
+
+* First release on PyPI.
```

Comment:
Good point, the cookiecutter itself won't; that was a copy-pasta from the output template.  Fixing.